engines/scumm/imuse_digi/dimuse_track.cpp

```diff
@@ -441,6 +441,7 @@ int IMuseDigital::transformVolumeLinearToEqualPow(int volume, int mode) {
 			break;
 		case 6:  // Half sine curve
 			eqPowValue = (1.0 - cos(mappedValue * M_PI)) / 2.0;
+			break;
```

Comment:
I don't know why this has been duplicated, it should't have. This is the same fix as commit 3d9a8862046acb84feb9bf6256c21b492cd67ff0